osgar/logger.py

```diff
@@ -229,24 +246,21 @@ def _create_index(data, pos, start_time=datetime.timedelta()):
     """
     index = []
     end = len(data)
-    times, prev = divmod(start_time, datetime.timedelta(microseconds=TIMESTAMP_OVERFLOW_STEP))
```

Comment:
I did not recognize the bits from older code, so maybe it is just "moving around"

Reply:
I don't understand this comment. Can you please clarify? Should I explain something?

Reply:
sorry, that was just my note, that these pieces I have seen in the new function ... closing.